Sources/NDT7Settings.swift

```diff
@@ -11,73 +11,40 @@ import Foundation
 /// Settings needed for NDT7.
 /// Can be used with default values: NDT7Settings()
 public struct NDT7Settings {
-
-    /// URL for Web Socket.
-    public var url: NDT7URL
+    public var allServers: [NDT7ServerV2]
+    
+    public var currentServerIndex: Int
+    
+    /// Configured URL to run downlaod test against
+    public var downloadUrl: String
+    
+    /// Configured URL to run upload test against
+    public var uploadUrl: String
+    
+    /// Hostname of configured NDT7 server
+    public var hostname: String
```

Comment:
If you have the urls already in allServers and the currentServerIndex used, you don't need to duplicate the information with these parameters.